accelerator/config.c

```diff
@@ -92,6 +93,8 @@ status_t ta_config_default_init(ta_config_t* const info,
 
   log_info(logger_id, "Initializing IRI configuration\n");
   tangle->milestone_depth = MILESTONE_DEPTH;
+  mkstemp(mss_tmp);
```

Comment:
Use `mkstemp' instead of `mktemp'.
Reference: https://lists.gnu.org/archive/html/bug-guile/2010-01/msg00042.html

Reply:
Right, just read that yesterday. This is `mkstemp` indeed.